README.md

```diff
@@ -53,53 +53,54 @@ import KlaviyoSwift
-
-let customerDictionary : NSMutableDictionary = NSMutableDictionary()
-customerDictionary[klaviyo.KLPersonEmailDictKey] = "john.smith@example.com"
-customerDictionary[klaviyo.KLPersonFirstNameDictKey] = "John"
-customerDictionary[klaviyo.KLPersonLastNameDictKey] = "Smith"
-
-let propertiesDictionary : NSMutableDictionary = NSMutableDictionary()
-propertiesDictionary["Total Price"] = 10.99
-propertiesDictionary["Items Purchased"] = ["Milk","Cheese", "Yogurt"]
-Klaviyo.sharedInstance.trackEvent(
-    eventName: "Completed Checkout",
-    customerProperties: customerDictionary,
-    properties: propertiesDictionary
-)
+let klaviyo = KlaviyoSDK()
+
+let event = Event(name: .StartedCheckout, properties: [
+    "Total Price": 10.99,
+    "Items Purchased": ["Milk","Cheese", "Yogurt"],
+], profile: [
+    "$first_name": "John",
+    "$last_name": "Smith",
+], value: 10.99)
+
+klaviyo.create(event: event)
 ```
 ### Arguments
 
-The `track` function can be called with up to four arguments.
+The `create` method takes an event object as an argument. The event can be constructed with the following arguments:
+
+* `name`: The name of the event you want to track, as a EventName enum.  The are a number of commonly used event names provided by default. If you need to log an event with a different name use `CustomEvent` with a string of your choosing. This argument is required to track an event.
 
-* `eventName`: The name of the event you want to track, as a string. This argument is required to track an event.
+* `profile`: An dictionary of properties that belong to the person who did the action you're tracking. Including an `$email`, `$phone_number` or `$id` key associates the event with particular profile in Klaviyo. In addition the SDK will retain these properties for use in future sdk calls, therefore if they are not included previously set identifiers will be used to log the event. This argument is optional.
 
-* `customerProperties`: An NSDictionary of properties that belong to the person who did the action you're tracking. If you do not include an `$email`, `$phone_number` or `$id key`, the event cannot be tracked by Klaviyo. This argument is optional but recommended.
+* `properties`: An dictionary of properties that are specific to the event. This argument is optional.
 
-* `properties`: An NSDictionary of properties that are specific to the event. This argument is optional.
+* `time`: This is the timestamp, as an `Date`, when the event occurred. This argument is optional but recommended if you are tracking past events. If you're tracking real- time activity, you can ignore this argument.
 
-* `eventDate`: This is the timestamp, as an NSDate, when the event occurred. This argument is optional but recommended if you are tracking past events. If you're tracking real- time activity, you can ignore this argument.
+* `value`: A numeric value (`Double`) to associate with this event. For example, the dollar amount of a purchase.
```

Comment:
Do we assume it's in dollars and not cents in any of our analytics calculations? Like calculating KAV etc?

Reply:
I think that's configurable maybe but pretty sure it's dollars.